packages/presets/src/ai/utils/event-source.ts

```diff
@@ -5,10 +5,12 @@ function delay(ms: number) {
 // do we really need to export this?
 export function toTextStream(
   eventSource: EventSource,
-  options?: {
+  options: {
     timeout?: number;
-  }
+    type?: string;
```

Comment:
why adding type here? I think the only event type not being listened to is `open`. There is no need to expose this to the outside

Reply:
Need to listen `attachment` event.

https://github.com/toeverything/AFFiNE/pull/6538/files#diff-e8a9e3cb567d0cc9ab5e33a5ad6cba230be641ad804f7b8440c852fcdbb98333R175